app/src/main/java/studio/lunabee/compose/accessibility/AccessibilityScreen.kt

```diff
@@ -0,0 +1,228 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import studio.lunabee.compose.R
+import studio.lunabee.compose.extension.topAppBarElevation
+import studio.lunabee.compose.lbcaccessibility.composable.AccessibilityCheckBoxRow
+import studio.lunabee.compose.lbcaccessibility.extension.addSemantics
+import studio.lunabee.compose.lbcaccessibility.model.AccessibilityDescription
+import studio.lunabee.compose.lbcaccessibility.model.StateDescription
+import studio.lunabee.compose.lbcaccessibility.state.AccessibilityState
+import studio.lunabee.compose.lbcaccessibility.state.rememberAccessibilityState
+import studio.lunabee.compose.lbctopappbar.material.LbcNavigationTopAppBar
+import studio.lunabee.compose.navigation.ToDirection
+
+@Composable
+fun AccessibilityScreen(
+    navigateToPreviousScreen: ToDirection,
+    openAccessibilitySettings: () -> Unit,
+) {
+    val lazyListState: LazyListState = rememberLazyListState()
+    val accessibilityState: AccessibilityState = rememberAccessibilityState()
+
+    Scaffold(
+        topBar = {
+            LbcNavigationTopAppBar(
+                title = stringResource(id = R.string.accessibility_title),
+                navigationIconRes = R.drawable.ic_back,
+                elevation = lazyListState.topAppBarElevation,
+                onNavigationClicked = navigateToPreviousScreen,
+                applyStatusBarPadding = true,
+                backgroundColor = MaterialTheme.colors.surface,
+                rowActionsComposable = {
+                    if (accessibilityState.isAccessibilityEnabled) {
+                        IconButton(onClick = openAccessibilitySettings) {
+                            Icon(
+                                painter = painterResource(id = R.drawable.ic_accessibility),
+                                contentDescription = stringResource(id = R.string.accessibility_screen_menu_content_description),
+                            )
+                        }
+                    }
+                }
+            )
+        },
+        floatingActionButton = {
+            if (!accessibilityState.isAccessibilityEnabled) {
+                FloatingActionButton(onClick = openAccessibilitySettings) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_accessibility),
+                        tint = MaterialTheme.colors.primary,
+                        contentDescription = null, // decorative, button will be moved to TopAppBar if talkback is activate.
+                    )
+                }
+            }
+        }
+    ) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/0da7e67017640b337b066a3410a30c89cff519b0/app/src/main/java/studio/lunabee/compose/accessibility/AccessibilityScreen.kt#L101"></span>Content padding parameter it is not used</td>
    </tr>
  </tbody>
</table>